3_visualize/src/3_plot.R

```diff
@@ -0,0 +1,45 @@
+# Create a plot
+create_plot <- function(data){
+  out_dir="personal/training/pipelines/ds-pipelines-targets-1/3_visualize/out/"
+  png(file = file.path(out_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
I think some of my previous comments can be applied here, as well. Your argument name makes sense here - what you are passing in is actual data. I would suggest adding a second argument for the png file that is created which specifies the filepath. You should be able to simplify your directory to just what is within your project, so it could look something like this: `create_plot <- function(data, out_file)` and when you share how to run it, you could write `create_plot(eval_data, '3_visualize/out/figure_1.png')`.

Reply:
agreed!